endorsement/storeapi.proto

```diff
@@ -43,6 +43,76 @@ message OpenArgs {
 
 message CloseArgs {
 }
+message SwComponentID {
+    common.TokenFormat type = 1;
+    google.protobuf.Struct id = 2;
+}
+// SwComponentInfo defines the structure of a single Software Component 
+// identified by a SoftwareComponent Identifer.
+message SwComponentInfo {
+    SwComponentID comp_id = 1;
+    google.protobuf.Struct comp_meta = 2; 
```

Comment:
we can simply call these `id` and `meta`

Reply:
There is already an "id" inside the SwComponentID, hence this way of representation!